CHANGELOG.md

```diff
@@ -391,7 +392,7 @@ to receive SIGTERM events, set the option `enableSigtermReporting = true`.
 
 ### Features
 
-The following two features, disabled by default, were mistakenly added to the release. We usually only add features in minor releases. 
+The following two features, disabled by default, were mistakenly added to the release. We usually only add features in minor releases.
```

Comment:
`l`: I would prefer having these fixes in an extra PR

Reply:
yes, make sense, I've missed this one.